src/main/java/com/dsl/expressions/bool/Contains.java

```diff
@@ -0,0 +1,14 @@
+package com.dsl.expressions.bool;
+
+import com.dsl.expressions.AbstractExpression;
+import com.dsl.expressions.Expression;
+import com.dsl.expressions.ExpressionGalore;
+import com.dsl.expressions.param.SelectorExpression;
+
+public class Contains extends AbstractExpression {
+
+    public Contains(final SelectorExpression e, final Expression... expressions) {
+        super(e, "CONTAINS", new ExpressionGalore(expressions));
```

Comment:
Why the array of exceptions as input? Shouldn't it be just one? Is it possible to pass multiple expressions to a contains clause?